MojioSDK/Client/AuthClient.swift

```diff
@@ -306,6 +306,10 @@ public class AuthClient: NSObject, AuthControllerDelegate {
                 failure(response: response.result.value as? NSDictionary)
             }
         }
```

Comment:
Let's take them out for the sake of simplicity. It makes it hard for 3rd party devs (even for us) when one has to dig into a bunch of settings to see why code is/isn't working.